tests/test_ingest_pipeline.py

```
@@ -0,0 +1,109 @@
+
+
+def test_clean_cached_results(tmp_path: Path) -> None:
+    cache_dir = tmp_path / "cache"
+    cache_dir.mkdir()
+
+    df1 = pd.DataFrame(
+        {
+            "file_prefix": ["run1"],
+            "doc_id": [1],
+            "metric": [0.1],
+            "some_path": ["/tmp/a"],
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/a"

(S108)